docs/api/rest_api/rest_api_reference/input/examples/user/groups/path/users/POST/UserCreate.xml.example

```diff
@@ -1,20 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <UserCreate>
   <mainLanguageCode>eng-GB</mainLanguageCode>
-  <remoteId>remoteId-qwert426</remoteId>
-  <login>yura</login>
-  <email>yurarajzer@example.net</email>
+  <remoteId>remoteId-23456789</remoteId>
+  <login>johnsmith</login>
+  <email>example@example.com</email>
   <password>Secrepassword5!</password>
-  <fields>
-    <field>
-      <fieldDefinitionIdentifier>first_name</fieldDefinitionIdentifier>
-      <languageCode>eng-GB</languageCode>
-      <fieldValue>Yura</fieldValue>
-    </field>
-    <field>
-      <fieldDefinitionIdentifier>last_name</fieldDefinitionIdentifier>
-      <languageCode>eng-GB</languageCode>
-      <fieldValue>Rajzer</fieldValue>
-    </field>
-  </fields>
-</UserCreate>
\ No newline at end of file
+		<fields>
+			<field>
+				<fieldDefinitionIdentifier>first_name</fieldDefinitionIdentifier>
+				<languageCode>eng-GB</languageCode>
+				<fieldValue>John</fieldValue>
+			</field>
+			<field>
+				<fieldDefinitionIdentifier>last_name</fieldDefinitionIdentifier>
+				<languageCode>eng-GB</languageCode>
+				<fieldValue>Smith</fieldValue>
+			</field>
+			<field>
+				<fieldDefinitionIdentifier>user_account</fieldDefinitionIdentifier>
+				<languageCode>eng-GB</languageCode>
+				<fieldValue>
+					<value key="hasStoredLogin">true</value>
+					<value key="contentId">14</value>
+					<value key="login">johnsmith</value>
+					<value key="email">example@example.com</value>
+					<value key="enabled">true</value>
+					<value key="maxLogin">10</value>
+				</fieldValue>
+			</field>
+	</fields>
```

Comment:
Formatting doesn't look good here. I think you have tabs instead of spaces. It should be always spaces and AFAICS we have here convention of 2 spaces indents.